test/app-tests/joining-test.tsx

```diff
@@ -57,7 +57,7 @@ describe('joining a room', function() {
 
         afterEach(function() {
             if (parentDiv) {
-                parentDiv.unmount();
+                createRoot(parentDiv!).unmount();
```

Comment:
I don't think this is right, that unmount would unmount the new root, maybe this line was faulty since the beginning.